dist/filter-menu/filter-menu.css

```diff
@@ -124,6 +117,24 @@ button.filter-menu__footer:hover,
 button.filter-menu-form__footer[type="submit"]:hover {
   background-color: var(--color-state-primary-hover, color-state-primary-hover);
 }
+button.filter-menu__footer:disabled,
+button.filter-menu-form__footer[type="submit"]:disabled,
+button.filter-menu__footer[aria-disabled="true"],
+button.filter-menu-form__footer[type="submit"][aria-disabled="true"] {
```

Comment:
So if it's type submit, it'll be a button. If it's not then it wont. So only for type submit we should have `:disabled` and not aria-disabled.